provisioning/device/src/Transports/Amqp/ProvisioningTransportHandlerAmqp.cs

```diff
@@ -28,39 +28,20 @@ public class ProvisioningTransportHandlerAmqp : ProvisioningTransportHandler
         /// <summary>
         /// Creates an instance of the ProvisioningTransportHandlerAmqp class using the specified fallback type.
         /// </summary>
-        /// <param name="transportFallbackType">The fallback type allowing direct or WebSocket connections.</param>
```

Comment:
I can't tell how the client supported "fallback" behavior, but I want to do away with this concept anyway (in accordance with the hub device client), so this is just specifying Tcp or web socket now.